packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -39,6 +40,12 @@ export type ColorRecipe = Recipe<Swatch>;
 /** @public */
 export type InteractiveColorRecipe = Recipe<InteractiveSwatchSet>;
 
+/** @public */
+export type InteractiveSet = Record<"rest" | "hover" | "active" | "focus", string>;
+
+/** @public */
+export type InteractiveRecipe = Recipe<InteractiveSet>;
```

Comment:
I don't see either of these types used anywhere. Are they needed?

Reply:
you are correct. I will remove these for now. thanks!